app/models/export/invoices/row.rb

```diff
@@ -15,23 +15,23 @@ def row_values
       end
 
       def customer_urn
-        MISSING
+        invoice.data['Customer URN'] || NOT_IN_DATA
```

Comment:
Not something that needs changing, but I quite like the the fetch-with-a-default syntax for this sort of thing: `invoice.data.fetch('Customer URN', NOT_IN_DATA)`

Reply:
Yes, and given that I've used that approach in `value_for` we might do well to do the same here.

Reply:
Although given that the changeover introduces the need for a mock, I might leave it for now. The to-do list looks too large for the next two days, so skipping the optionals for the time being.

Reply:
Change made